packages/editor-ui/src/components/Sticky.vue

```diff
@@ -410,4 +403,32 @@ export default defineComponent({
 		cursor: pointer;
 	}
 }
+
+.sticky-color-1 {
```

Comment:
why are we adding classes here? and in sticky component in design system?

Reply:
Because we are not passing the exact color to the component but the index. We need the color index in both places. Outside the component to render the color palette and inside the render the proper background color.

Reply:
ah ya sorry obviously 😅